.github/workflows/ci-cd-main.yml

```diff
@@ -108,9 +108,18 @@ jobs:
       version: ${{ needs.get-current-version.outputs.version }}-${{ needs.generate-git-short-sha.outputs.gitShortSha }}
       runMigration: ${{ github.event_name == 'workflow_dispatch' || needs.check-for-changes.outputs.hasMigrationChanges == 'true' }}
 
+  publish-nuget:
```

Comment:
`publish-sdk-to-nuget` så vi synliggjør her at vi publiserer SDK kun. Kanskje vi har lyst til å publisere flere nugets etterhvert